docs/dev/fundamentals/reading-game-code.md

```diff
@@ -2,96 +2,122 @@
 ---
 
-# Reading game's code
+# Reading Game Code
 ## Introduction
 Chances are, early on in your modding journey you will be met by quite a conundrum - You want to know how something in the game works, but there is an issue! You cannot just open the game's files to find it *(you silly goose)*. 
 
 You see, the way Unity games work, is that their code gets compiled into a .dll file, which you cannot normally open. 
 
-Fortunately, there is something called a "decompiler". What it does, is it grabs game's code and reverses the process, resulting in readable code. While it is not perfect, it gets it's job done.
+## Decompilers
+Fortunately, there is something called a "decompiler". What it does, is it grabs the game's code and reverses the process, resulting in readable code. While it is not perfect, it gets it's job done.
 
-::: warning
-Please keep in mind that while decompiling and reading the code is *not* illegal, distributing said decompiled code *is* illegal. Same applies for sharing the game's files. 
+Currently there are 3 decompilers that are widely used:
 
-What this means is that you **CANNOT** bundle any of the game's files with your mods.
-:::
+[dnSpyEx](https://github.com/dnSpyEx/dnSpy), [ILSpy](https://github.com/icsharpcode/ILSpy) and [dotPeek](https://www.jetbrains.com/decompiler/). 
 
-## ILSpy
-[ILSpy](https://github.com/icsharpcode/ILSpy/) is an open-source .NET decompiler.
+::: info
+Picking which decompiler to use is up to personal preference, but for this article ILSpy will be used due to it's balance between simplicity and power.
+:::
 
-Downloading it is pretty simple, go to the [Releases](https://github.com/icsharpcode/ILSpy/releases/latest) page and download an installer that matches you system. 
+::: danger
+Please keep in mind that while decompiling and reading the code is *not* illegal, distributing said decompiled code *is* illegal. Same applies for sharing the game's files. 
 
-::: tip
-Apart from ILSpy, there is another popular decompiler called "dnSpyEx". Picking which one to use is up to personal preference, but for this article ILSpy will be used due to it's simplicity.
+What this means is that you **CANNOT** bundle any of the game's files with your mods.
 :::
 
 ## Opening the game DLL
 Upon starting ILSpy you will be met with a window similar to this.
-![Screenshot of empty ILSpy window](/images/reading-games-code/ilspy-window.png)
+![Screenshot of empty ILSpy window](/images/reading-game-code/ilspy-window.png)
 
 From then you want to open the `Assembly-CSharp.dll` file present in the game's files. 
 
 To get it you need to do the following:
-1. Open game files. On Steam it's done as simple as right clicking the game and pressing 'Browse local files'.
-![How to open local files on Steam](/images/reading-games-code/opening-local-files.png)
+### 1. Open game files. On Steam it's done as simple as right clicking the game and pressing 'Browse local files'.
```

Comment:
If you notice the "anchor" that is added to the url when you click on this, you can see it's very long:

`/dev/fundamentals/reading-game-code#_1-open-game-files-on-steam-it-s-done-as-simple-as-right-clicking-the-game-and-pressing-browse-local-files`

To fix this, you can simply add {#new-anchor} after the end of the heading to create a different anchor:

`/dev/fundamentals/reading-game-code#new-anchor`

The markdown file then looks like this:
```md
## Really long anchor with lots of text & symbols! {#short-anchor}
```